util/fibers/stacktrace.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023, Roman Gershman.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include "util/fibers/stacktrace.h"
+
+#include <dlfcn.h>
+#include <execinfo.h>
+
+#include "absl/strings/str_format.h"
+#include "absl/strings/str_join.h"
+#include "base/logging.h"
+
+std::string util::GetStacktrace(bool symbolize) {
+  void* addresses[20];
+  int size = backtrace(addresses, sizeof(addresses) / sizeof(void*));
```

Comment:
I don't know if `backtrace()` is portable (like on MacOS / FreeBSD), maybe use [`absl::GetStackTraceWithContext()`](https://github.com/abseil/abseil-cpp/blob/master/absl/debugging/stacktrace.h) instead?